homeassistant/components/onvif/event.py

```diff
@@ -112,6 +156,14 @@ async def async_renew(self) -> None:
         )
         await self._subscription.Renew(termination_time)
 
+    def schedule_pull(self) -> None:
```

Comment:
Please rename to `async_schedule_pull`. Our standard is to name all functions that should run in the event loop `async_`. Not only coroutine functions.